public/survey.css

```diff
@@ -1,3 +1,168 @@
 div.sv_qstn_error_top  span.glyphicon-exclamation-sign {
     display: none
-}
\ No newline at end of file
+}
+.progress {
+  height: 23px !important;
+}
+
+.progress-bar {
+  background-color: #2572b4 !important;
+}
+.checkbox label, .radio label {
+  padding-left: 30px !important;
+}
+
+.form-control {
+  font-size: 2rem !important;
+}
+
+.btn-default {
+  color: #335075 !important;
+  background-color: #eaebed !important;
+  border-color: #dcdee1 !important;
+}
+
+.container-fluid {
+  padding-right: 15px !important;
+  padding-left: 15px !important;
+  margin-right: auto !important;
+  margin-left: auto !important;
+}
+
+.h1, .h2, .h3, .h4, .h5, .h6, h1, h2, h3, h4, h5, h6 {
```

Comment:
Issues seems wot bhe with the changes to headers somehow.

Reply:
Okay - I'm glad you've resolved the floating issue.

Reply:
Was rushing to get the kids, haven't had the chance to fix it yet lol :grin:

Reply:
Ah - I have one too (he just turned 5 and started Kindergarten!) I thought you wanted me to fix this... So I'll get out of your way. Just let me know if you want me to do anything.

Reply:
No, no, if you have a few cycles, you're way better than me at CSS just from looking at the magic you've done :grinning:

Reply:
So, as far as I can tell, the issue is with h2, which seems to be used throught the [WET](https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/web-experience-toolkit.html) template via the script for screen reader content.

By changing the behaviour of the CSS for our own h2 headers, it looks like it interferes with the injected HTML. At least that's what I think is happening :thinking:

Reply:
My only workaround for now would be to remove h2 as an element to be included on line 32.

If you have another option, please go ahead. If not, if you could please fix and push your commit, I'll merge the PR and we'll have a new UI for the score! :grinning: 

Thanks so much for all the great work you've done on this project!

Reply:
Okay - I'm on it.

Reply:
The only problem is I can't reproduce what you're seeing. 
I'm using Chrome Version 77. This is what I get when I hit the 'Start Again' button.

![image](https://user-images.githubusercontent.com/20051/65366149-0cc0f500-dbd5-11e9-9ebb-0cf21179ba7b.png)

Reply:
Your image above seems to suggest you have scrollbars appearing in the window. Could you try Window|Zoom? See if that makes a difference? Also, of course clear your cache. You can also try a Empty Cache and Hard Reload, but you need the Inspector open to see that after holding down the reload button.

![image](https://user-images.githubusercontent.com/20051/65366184-907ae180-dbd5-11e9-99a2-c882826a7a93.png)

Reply:
Safari v13
![image](https://user-images.githubusercontent.com/20051/65366245-37f81400-dbd6-11e9-8566-dd043d2964c1.png)

Firefox v69
![image](https://user-images.githubusercontent.com/20051/65366258-45ad9980-dbd6-11e9-820e-04c5f418ca9e.png)

Reply:
OK, testing it out!

Reply:
Hmm, so I tested on Chrome 77, both on Windows 10 Pro and Ubuntu 18.04 and both behave the same. The bottom scroll bar is there for Chrome but not other browsers 🤔 

Behaviour is as expected with Edge 44 and Firefox 69.0.1. This is really odd... Will try on a MacOS device to see what's going on.

Reply:
![image](https://user-images.githubusercontent.com/1690085/65374797-0590fa00-dc5c-11e9-9c77-fba79db17d94.png)

Reply:
I can now reproduce the scrollbars you mention in Chrome, I think. But tell me are you able to scroll to the left and show the page correctly?

Reply:
I've followed your suggestion and it works on my Mac.

![image](https://user-images.githubusercontent.com/20051/65380786-b072d980-dc97-11e9-9bae-f8dbaac47ee7.png)